### PR TITLE
Minor fixes and amendments to Getting Started

### PR DESCRIPTION
**Pull request by chris-morgan**
2½ fixes and 1²⁄₂ more subjective matters which you’re quite at liberty to reject. Glommed together for my own convenience, really.

### CODE REVIEW

**Comment by sgrif** [comment]
Can you submit the code changes to https://github.com/sgrif/diesel_demo first?

File: source/views/guides/getting-started.html.slim [hunk 1]
```diff
@@ -50,10 +50,10 @@ main
             pre.demo__example-snippet
               code
                 | [dependencies]
-                  diesel = "0.6.1"
-                  diesel_codegen = { version = "0.6.1", default-features = false, features = ["nightly", "postgres"] }
-                  dotenv = "0.8.0"
-                  dotenv_macros = "0.8.0"
+                  diesel = "0.6"
```
Comment (sgrif) [inline source/views/guides/getting-started.html.slim]: I don't think this is the same, is it? `diesel = "0.6"` is the same as `diesel = "0.*"` as far as I'm aware.

Comment (chris-morgan) [inline source/views/guides/getting-started.html.slim]: 1.6 would be `>1.6.0,<2.0.0`, but pre-1.0.0 it’s different; 0.6 is `0.6.*`.

Comment (sgrif) [inline source/views/guides/getting-started.html.slim]: I think it's probably better to leave this as is. While there won't be any major breaking changes in minor versions post 1.0, there may still be "minor breaking changes" that could cause code to fail to compile, and are allowed in minor versions as per the [API evolution guidelines](https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md).

**Comment by sgrif** [comment]
Most of these changes have made it in through other means.

**Pull request closed without merging**